core/src/main/java/org/commonjava/maven/ext/manip/state/DependencyState.java

```diff
@@ -96,7 +101,13 @@ public DependencyState( final Properties userProps )
         ignoreSuffix = Boolean.valueOf( userProps.getProperty( STRICT_ALIGNMENT_IGNORE_SUFFIX, "false" ) );
         failOnStrictViolation = Boolean.valueOf( userProps.getProperty( STRICT_VIOLATION_FAILS, "false" ) );
         remoteBOMdepMgmt = IdUtils.parseGAVs( userProps.getProperty( DEPENDENCY_MANAGEMENT_POM_PROPERTY ) );
+
         dependencyExclusions = PropertiesUtils.getPropertiesByPrefix( userProps, DEPENDENCY_EXCLUSION_PREFIX );
+        // If dependencyExclusions have not been set via exclusion prefix attempt to set it via the override alias prefix.
+        if ( dependencyExclusions.isEmpty())
+        {
+            dependencyExclusions = PropertiesUtils.getPropertiesByPrefix( userProps, DEPENDENCY_OVERRIDE_PREFIX );
```

Comment:
If dependencyOverride and dependencyExclusion are aliases, then I would expect them to be used interchangeably. If there are clashes between them, then I would flag that as an error, or some explicit overridable warning if that is indeed intended, so as to avoid any confusion about which one wins, or doing it by accident.